### PR TITLE
 Change Pulp 3 Default Ports

### PR DESCRIPTION
**Pull request by mikedep333**
Change Pulp 3 Default Ports
https://pulp.plan.io/issues/4556

Update docs for changed Pulp 3 Default Ports
https://pulp.plan.io/issues/4594

Content: 8080 -> 24816
API: 8000 -> 24817

This commit includes a change to docs/Makefile that affects the build process.

Note: I am about to submit changes to ansible-pulp & several other repos too. The travis files exist in those other repos in addition to doc updates. They probably all need to be merged at the same time.

Required PR: https://github.com/pulp/pulpcore/pull/79

### CODE REVIEW

File: .travis/script.sh [hunk 1]
```diff
@@ -21,7 +21,7 @@ wait_for_pulp() {
 }
 
 if [ "$TEST" = 'docs' ]; then
-  django-admin runserver >> ~/django_runserver.log 2>&1 &
+  django-admin runserver 24817 >> ~/django_runserver.log 2>&1 &
   sleep 5
```
Comment (dkliban) [inline .travis/script.sh]: try increasing this sleep to 10 seconds.

Comment (mikedep333) [reply .travis/script.sh]: Thank you for your help in debugging this.

I created https://github.com/pulp/pulpcore/pull/79 and rebased this PR/branch onto it.

No additional sleep was needed.

File: pulpcore/app/viewsets/repository.py [hunk 2]
```diff
@@ -170,7 +170,7 @@ class RepositoryVersionFilter(BaseFilterSet):
     # /?number__range=4,6
     # /?_created__gte=2018-04-12T19:45
     # /?_created__range=2018-04-12T19:45,2018-04-13T20:00
-    # /?content=http://localhost:8000/pulp/api/v3/content/file/fb8ad2d0-03a8-4e36-a209-77763d4ed16c/
+    # /?content=http://localhost:24817/pulp/api/v3/content/file/fb8ad2d0-03a8-4e36-a209-77763d4ed16c/
```
Comment (dkliban) [inline pulpcore/app/viewsets/repository.py]: this needs to be just the relative URL without the hostname and port. What was here before was wrong.

Comment (mikedep333) [reply pulpcore/app/viewsets/repository.py]: @dkliban Fixed.

**Pull request merged**